generator/internal/dart/annotate.go

```diff
@@ -620,7 +620,8 @@ func (annotate *annotateModel) createFromJsonLine(field *api.Field, state *api.A
 		default:
 			return fmt.Sprintf("decodeMap(%s)%s", data, bang)
 		}
-	case field.Typez == api.INT64_TYPE || field.Typez == api.UINT64_TYPE:
+	case field.Typez == api.INT64_TYPE || field.Typez == api.UINT64_TYPE || field.Typez == api.SINT64_TYPE ||
```

Comment:
nit: I wonder if we could add a helper function to make this more readable and/or break this into lines:

```suggestion
	case field.Typez == api.INT64_TYPE || 
	    field.Typez == api.UINT64_TYPE || 
	    field.Typez == api.SINT64_TYPE ||
```

Reply:
👍 , this reads better